NorthstarDedicatedTest/serverauthentication.cpp

```diff
@@ -511,7 +515,7 @@ char CGameClient__ExecuteStringCommandHook(void* self, uint32_t unknown, const c
 	ConCommand* command = g_pCVar->FindCommand(tempCommand.Arg(0));
 
 	// if the command doesn't exist pass it on to ExecuteStringCommand for script clientcommands and stuff
-	if (command && !command->IsFlagSet(FCVAR_CLIENTCMD_CAN_EXECUTE))
+	if (command && !command->IsFlagSet(FCVAR_CLIENTCMD_CAN_EXECUTE) && !CGameClient__IsEngineClientCommand(NULL, tempCommand))
```

Comment:
It would probably be better and safer to try adding the flag `FCVAR_CLIENTCMD_CAN_EXECUTE` to the `status` command itself rather than this

Reply:
From @rolelessweapon on Discord:

> The same CGameClient::IsEngineClientCommand check exists in other Source engine games.
> You'd also have to add the `FCVAR_CLIENT_CAN_EXECUTE` command to ping as well, which would work, but feels clunky rather than reproducing the existing check

https://discord.com/channels/920776187884732556/950322078945538058/986267580526374952